### ISSUE TITLE
Bug with pjax and DataColumn

### ISSUE THREAD

**Issue by stepancher** [kartik-v/yii2-grid#434]
If i don't use Pjax, i have error "Undefined index: options" in line 355 in DataColumn. Please add check in this file like this 
```      
      if (Config::isInputWidget($this->filterType) && $this->grid->pjax) {
                    $options['pjaxContainerId'] = $this->grid->pjaxSettings['options'][' 
      }
```

**Comment by Imam86** [comment]
Yep. After updated through composer today, I got this error, too.
Thanks for the solution. So I changed DataColumn.php at line 354.

Before:
```php
if (Config::isInputWidget($this->filterType)) {
```
After:
```php
if (Config::isInputWidget($this->filterType) && $this->grid->pjax) {
```
And the error is gone.

**Comment by kartik-v** [comment]
Closed via [update to release v3.1.0](https://github.com/kartik-v/yii2-grid/commit/8917f5729852770622ba9e2b92c141ac44b2413f).

**Comment by enscope** [comment]
When to expect 3.1.0 be available for composer? "dev" branch (and composer packagist) both point to 3.0.9 as most recent version, which contains this bug. Thank you!

**Comment by unNene** [comment]
@kartik-v please DataColumn.php version 3.1.0, line 354.
https://github.com/kartik-v/yii2-grid/blob/8917f5729852770622ba9e2b92c141ac44b2413f/DataColumn.php

There is a error, you have to use $this->grid->pjax instead of $this->pjax

**Comment by kartik-v** [comment]
Duplicate to and fixed via #438.

**Comment by enscope** [comment]
Is someone reading me here? What I was asking is, if there will be 3.1.0 published on packagist as well or we are stuck in using dev-master?

Ref: https://packagist.org/packages/kartik-v/yii2-grid

**Comment by dogrocker** [comment]
Same as @enscope .
I stuck with 3.0.9 too got an error.

**Comment by kartik-v** [comment]
The response was for query by @unNene - the bug is already fixed in v3.1.0 

@enscope @dogrocker in response to your query - the release v3.1.0 is not tagged yet and will be done a bit later after some tests.

**Comment by dogrocker** [comment]
Thank you very much @kartik-v.
